web/template/mlpa-history.gohtml

```diff
@@ -3,15 +3,81 @@
 {{ define "title" }}History{{ end }}
 
 {{ define "main" }}
+  {{ template "mlpa-header" (caseTabs .CaseSummary "history") }}
 
-{{ template "mlpa-header" (caseTabs .CaseSummary "history") }}
-
-<div class="moj-page-header-actions">
+  <div class="moj-page-header-actions">
     <div class="moj-page-header-actions__title">
-        <h1 class="govuk-heading-l">History</h1>
+      <h1 class="govuk-heading-l">History</h1>
+    </div>
+  </div>
+  {{ range .EventData }}
```

Comment:
Needs a wrapper to show the left-hand bar
```suggestion
  <div class="moj-timeline">
  {{ range .EventData }}
```

Reply:
so it does